File: CompSci 1/Unit 8, String Class/Acronym/Acronym.java
import java.util.Scanner;

public class Acronym {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter three words: ");
        String word1 = sc.next().substring(0,1);
        String word2 = sc.next().substring(0,1);
        String word3 = sc.next().substring(0,1);


        System.out.println("Your acronym is: " + word1 + word2 + word3);
    }
}

File: CompSci 1/Unit 8, String Class/Acronym/Acronym.kt
import java.util.Scanner

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    print("Enter three words: ")
    val word1 = sc.next().substring(0, 1)
    val word2 = sc.next().substring(0, 1)
    val word3 = sc.next().substring(0, 1)


    println("Your acronym is: $word1$word2$word3")
}